Sources/Commands/SwiftBuildTool.swift

```diff
@@ -45,7 +45,8 @@ public class SwiftBuildTool: SwiftTool<BuildToolOptions> {
 
     override class func defineArguments(parser: ArgumentParser, binder: ArgumentBinder<BuildToolOptions>) {
         binder.bind(
-            option: parser.add(option: "--build-tests", kind: Bool.self),
+            option: parser.add(option: "--build-tests", kind: Bool.self,
+                usage: "Build the test classes without running them"),
```

Comment:
I wonder if we should clarify that this both builds all the products *and* the tests? Not sure if that is concise enough and maybe it isn't confusing?

In any case +1 to the improvement, thanks!